partner_relations/model/res_partner.py

```diff
@@ -298,12 +298,11 @@ def search(self, cr, uid, args, offset=0, limit=None, order=None,
             cr, uid, args + date_args + active_args, offset=offset,
             limit=limit, order=order, context=context, count=count)
 
-    def read(
-            self, cr, uid, ids, fields=None, context=None,
-            load='_classic_read'):
-        return super(ResPartner, self).read(
-            cr, uid, ids, fields=fields,
-            context=self._update_context(context, ids), load=load)
+    @api.multi
+    def read(self, fields=None, load='_classic_read'):
+        self.env.context = self._update_context(
+            self.env.context, [r.id for r in self])
+        return super(ResPartner, self).read(fields=fields, load=load)
```

Comment:
I already did. I found that the difference between when ```partner_relations``` is installed and when it is not is, in first case, ```FailedValue``` object is instanciated with ```AccessError``` exception. While, in second case, it is instanciated with ```MissingError``` exception.
The correct behaviour is to have a ```MissingError``` exception, because it is later correctly intercepted by ORM. Instead, ```AccessError``` raises the error pointed out by https://github.com/OCA/partner-contact/issues/154
Thus I noticed it was something related to ```read``` method and tried rewriting it with new API.
I don't have additional information at the moment.

@hbrunn maybe you can go further with debugging?